api/v1alpha1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1 contains API Schema definitions for the operators v1 API group
-//+kubebuilder:object:generate=true
-//+groupName=operators.io.operator-framework
+// +kubebuilder:object:generate=true
+// +groupName=operators.io.operator-framework
```

Comment:
I am not sure, I did `make verify` command and it updated above file.

Reply:
Yeah this seems to be something with `go fmt` with Go 1.19 so if you are running Go 1.19 it will update it as part of running `make verify`.

I think this change is fine as I imagine we will probably attempt to move this to use Go 1.19 sooner rather than later.